### ISSUE TITLE
Data missing intermittently

### ISSUE THREAD

**Issue by reynoldsnlp** [BYU-ODH/yvideo-client#840]
@ThCovenant reported the following:

> Second, we are experiencing something weird where in working in Y-video for a bit, it suddenly decides that there are no collections for a particular professor.  Any idea why that is happening?

**Comment by reynoldsnlp** [comment]
Need more information.

**Comment by ThCovenant** [comment]
Certainly.  Please view the attached video of the process.  While viewing one professor's collections, attempting to move to another professor's collections generates a message that the second professor has no collections (when he actually does).  Searching for the first professor's collections generates the same message.  We discovered that refreshing the page reveals the collections correctly.
________________________________

**Comment by reynoldsnlp** [comment]
[This video](https://user-images.githubusercontent.com/13379223/214142043-77a033bb-3ca3-482b-9e9f-db79252e2644.mp4) demonstrates the unwanted behavior.
